frontend/src/components/header/mobileMenu/MobileMenu.tsx

```diff
@@ -29,11 +37,16 @@ const MobileMenu: React.FC<MobileMenuProps> = ({ anchorEl, handleCloseMenu, hand
                 onClose={handleCloseMenu}
                 className="settings-menu"
             >
-                {HEADER_MAIN_TABS.map((page) => (
+                {loggedUser.userType === UserType.Customer ? HEADER_CLIENT_MAIN_TABS.map((page) => (
+                    <MenuItem key={page} onClick={handleCloseMenu}>
+                        <Typography className="typography">{page}</Typography>
+                    </MenuItem>
+                )) : HEADER_MERCHANT_MAIN_TABS.map((page) => (
```

Comment:
It makes sense to pass tabs from header component to not duplicate logic here and in desctopMenu compoennt